scripts/make_release.sh

```diff
@@ -39,6 +39,41 @@ tag_cabal_ver_re() {
   tag_cabal_ver "$1" | sed -e 's/\./\\./g'
 }
 
+################################################################################
+# Interactively change the release-specific parameter by promting the caller, and
+# mutating the script itself.
+
+echo "Previous release: $GIT_TAG"
+new_tag=$(date +v%Y-%m-%d)
+read -e -p "New release tag: " -i "$new_tag" new_tag
+
+SCRIPT=`realpath $0`
+sed -i -e "s/^OLD_GIT_TAG=\"$OLD_GIT_TAG\"/OLD_GIT_TAG=\"$GIT_TAG\"/g" $SCRIPT
+sed -i -e "s/^GIT_TAG=\"$GIT_TAG\"/GIT_TAG=\"$new_tag\"/g" $SCRIPT
+
+OLD_GIT_TAG=$GIT_TAG
+GIT_TAG=$new_tag
+
+OLD_CARDANO_NODE_TAG=$CARDANO_NODE_TAG
+read -e -p "Cardano node tag: " -i "$CARDANO_NODE_TAG" CARDANO_NODE_TAG
```

Comment:
I don't like scripts which ask questions. CLI args are better, something like:
```
usage: ./scripts/make_release.sh ( today | vYYYY-MM-DD ) [CARDANO_NODE_TAG]
```

Reply:
Normally I'd agree with you, but here I'm after interactivity and a low learning curve.

This way the user gets provided sensible defaults, but has the responsibility to review the parameters.

If you want to make a release tag for tomorrow, you can simply change the last digit of the default tag. You don't have to make the full jump from the automatic `today` to the manual `v2021-08-24`.